README.md

```diff
@@ -363,6 +363,47 @@ function router(app) {
 }
 ```
 
+## Email Addresses validation
+
+Please check Mailgun [email validation documentation](https://documentation.mailgun.com/api-email-validation.html) for responses details.
+
```

Comment:
Can we add a note specifying that these routes require the Mailgun public API key please.